src/main/java/org/folio/rest/utils/ExpirationTool.java

```diff
@@ -123,11 +122,10 @@ public static void doExpiration(Vertx vertx, Context context) {
     context.runOnContext(v -> {
       try {
         PostgresClient pgClient = PostgresClient.getInstance(vertx, tenant);
-        pgClient.setIdField("id");
         Criteria idCrit = new Criteria();
         idCrit.addField(USER_ID_FIELD);
         idCrit.setOperation("=");
-        idCrit.setValue(user.getId());
+        idCrit.setVal(user.getId());
```

Comment:
Can you replace the complete saveUser method by a PgUtil.put invokation so that this duplicated code gets removed?